app/components/user_avatars_stack/users_list/row.tsx

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import withObservables from '@nozbe/with-observables';
+import React, {useCallback} from 'react';
+import {Text, TouchableOpacity, View} from 'react-native';
+
+import FormattedText from '@components/formatted_text';
+import ProfilePicture from '@components/profile_picture';
+import {preventDoubleTap} from '@utils/tap';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {displayUsername} from '@utils/user';
+
+import type UserModel from '@typings/database/models/servers/user';
+
+type Props = {
+    currentUserId: string;
+    teammateNameDisplay: string;
+    theme: Theme;
+    user: UserModel;
+}
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
+    return {
+        container: {
+            flexDirection: 'row',
+            justifyContent: 'flex-start',
+            height: 40,
+            width: '100%',
+            alignItems: 'center',
+        },
+        profile: {
+            alignItems: 'center',
+            justifyContent: 'center',
+            borderWidth: 1,
+            borderRadius: 12.5,
+            width: 25,
+            height: 25,
+            borderColor: changeOpacity(theme.centerChannelColor, 0.08),
+        },
+        textContainer: {
+            flexDirection: 'row',
+            marginLeft: 12,
+            marginRight: 24,
+        },
+        displayName: {
+            fontSize: 15,
+            fontWeight: '400',
+            color: theme.centerChannelColor,
+        },
+        username: {
+            fontSize: 15,
+            fontWeight: '400',
+            color: changeOpacity(theme.centerChannelColor, 0.56),
+        },
+    };
+});
+
+const Row = ({currentUserId, teammateNameDisplay, theme, user}: Props) => {
+    const goToUserProfile = useCallback(preventDoubleTap(async () => {
+        // @todo: Navigate to User Profile
+    }), []);
+
+    if (!user.id) {
+        return null;
+    }
+
+    const style = getStyleSheet(theme);
+
+    const {id, username} = user;
+    const usernameDisplay = '@' + username;
+    const displayName = displayUsername(user, undefined, teammateNameDisplay);
+    const isCurrentUser = currentUserId === id;
+    return (
```

Comment:
For user search and user autocomplete we already use some rows to show users. We may consider unifying those with this one. 0/5.